.pylintrc

```diff
@@ -41,7 +41,7 @@ extension-pkg-whitelist=
 # Pylint and this flag can prevent that. It has one side effect, the resulting
 # AST will be different than the one from reality. This option is deprecated
 # and it will be removed in Pylint 2.0.
-optimize-ast=no
+#optimize-ast=no
```

Comment:
How comes this changed?

Reply:
Generated new .pylintrc and updated any overides we had